pkg/cmd/create/create.go

```diff
@@ -463,12 +475,9 @@ func (o *Options) findPreviewURL(envVars map[string]string) (string, error) {
 
 	log.Logger().Infof("found helm preview release %s in namespace %s", info(releaseName), info(releaseNamespace))
 
-	application := o.Repository
-	app := naming.ToValidName(application)
-
 	appNames := []string{o.PreviewService}
 	if o.PreviewService == "" {
-		appNames = []string{app, releaseName, o.Namespace + "-preview", releaseName + "-" + app}
+		appNames, _ = findAllServiceNamesInNamespace(o.KubeClient, releaseNamespace)
```

Comment:
Why skip error check?

Reply:
I should probably return "", errors.Wrapf(err,"finding services in the preview namespace")

Reply:
use fmt.Errorf, errors package is deprecated and not maintained anymore. Atleast for new code we should avoid using it.

Reply:
Sure